tests/models/cohere/test_modeling_cohere.py

```diff
@@ -373,10 +381,10 @@ def test_batched_4bit(self):
 
         EXPECTED_TEXT = [
             'Hello today I am going to show you how to make a simple and easy card using the new stamp set called "Hello" from the Occasions catalog. This set is so versatile and can be used for many occasions. I used the new In',
-            "Hi there, here we are again with another great collection of free fonts. This time we have gathered 10 free fonts that you can download and use in your designs. These fonts are free for personal and commercial use. So",
+            "Hi there, here we are again with another great collection of free fonts for your next project. This time we have gathered 10 free fonts that you can download and use in your designs. These fonts are perfect for any kind",
         ]
 
-        model = CohereForCausalLM.from_pretrained(model_id)
+        model = CohereForCausalLM.from_pretrained(model_id, device_map="auto")
```

Comment:
This was previously GPU OOM. After changing to `auto`, I have to update the expected output value too.